library/stats.py

```diff
@@ -532,3 +571,195 @@ def stats():
                                                THEME_DATA['STATS']['DISK']['FREE']['TEXT'].get("BACKGROUND_IMAGE",
                                                                                                None))
             )
+
+
+class Net:
+    @staticmethod
+    def stats():
+        pnic_before_file = "/tmp/pnic_before"
```

Comment:
Is it possible to avoid using an external file?

Reply:
Sorry, I haven't found how to do it yet.

Reply:
Found how to do without temporary file.
I pushed the commit (below).